check_run/check_run/doctype/check_run/check_run.py

```diff
@@ -518,7 +518,7 @@ def render_check_pdf(self: Self, reprint_check_number: int | None = None) -> Non
 				0,
 			)
 		frappe.db.commit()
-		frappe.publish_realtime("reload", "{}", doctype=self.doctype, docname=self.name)
+		# frappe.publish_realtime("reload", "{}", doctype=self.doctype, docname=self.name)
```

Comment:
Whoops

Reply:
I thought it was intentional 🤦🏻

Reply:
No and it covers up the original problem. Without it, it's half a fix.